SpatialGDK/Build/Scripts/BuildWorker.bat

```diff
@@ -1,22 +1,73 @@
 @echo off
 
-rem Generate worker configs
-pushd "%~dp0..\..\..\..\..\..\spatial"
-spatial worker build build-config
-popd
+if [%4] == [] goto :MissingParams
 
-rem Build Unreal project
-pushd "%~dp0..\..\..\..\..\"
+rem Try and build as a project plugin first, check for a project plugin structure.
+set UnrealProjectDir="%~dp0..\..\..\..\..\"

```

Comment:
Variable naming style is inconsistent (`%UnrealProjectDir%` vs `%BUILD_EXE_PATH%`)